Jenkinsfile

```diff
@@ -64,6 +64,38 @@ install_autogluon = """
     python3 -m pip install --upgrade -e autogluon/
 """
 
+stage("Lint Check") {
+  parallel 'mm': {
```

Comment:
Is it better to change `mm` -> `multimodal`? We will use name `multimodal` for the new name space.